tsdoc/src/details/StandardTags.ts

```diff
@@ -124,6 +124,18 @@ export class StandardTags {
    * to the other item, which may be an unrelated class, or even an import from a
    * separate NPM package.
    *
+   * What gets copied
+   * 
+   * The @inheritDoc tag does not copy the entire comment body. Only the following 
```

Comment:
You need to quote these tags using backticks, otherwise they will be interpreted as actual TSDoc tags:

Like this:
```
* The `@inheritDoc` tag does not copy the entire comment body. Only the following 
```